charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -364,7 +370,7 @@ spec:
               port: {{ if .Values.api.tls.serveHTTPS }}https{{ else }}http{{ end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-          {{- if or .Values.api.tls.serveHTTPS .Values.global.enterpriseLicenseSecretRef .Values.global.customCaSecretRef }}
+          {{- if or .Values.api.tls.serveHTTPS .Values.global.enterpriseLicenseSecretRef .Values.global.customCaSecretRef .Values.api.features.bootstrapConfig.enabled }}
```

Comment:
I do not understand why this extra predicate was added as there is no additional `volumeMounts` below.

Reply:
This is a bug fix, there in fact is a volume mount for bootstrap config

Reply:
Ok.